### PR TITLE
JavaScript Standard Style

### PR DESCRIPTION
**Pull request by ghost**
This is coming in in a few commits. Here's what to expect:

- [x] Add Standard with NPM script
- [x] Run fixable linters to mitigate human error
- [x] Sweep test
- [x] Resolve remaining linters
- [x] Regression test
- [x] Once regressed, add git hooks and run NPM script on pre-commit <del>and CI</del>
- [x] Merge and enjoy

### CODE REVIEW

**Comment by ghost** [comment]
@eduardoboucas this is in your court now. please let me know when I'm good to continue.

**Comment by eduardoboucas** [comment]
@jhabdas Looks super! The only thing I'm torn about is using single quotes for static props in JSX. It it possible to add something like [this](http://eslint.org/docs/rules/jsx-quotes#prefer-double) to Standard?

**Comment by ghost** [comment]
@eduardoboucas I hear you on that. When I first started writing JSX I tried to make everyone use double-quotes in my org. It was the old dog in me. After spending some time with a few ace React devs though I came around. Here's why single quotes:

- Because JSX is really JavaScript
- Single quotes are standard in React Native
- Single quotes require less keystrokes, resulting in less fatigue

As for updating the linter, the answer is [maybe](https://github.com/feross/eslint-config-standard-react/issues/36) and [no](https://standardjs.com/#i-disagree-with-rule-x-can-you-change-it).

**Comment by eduardoboucas** [comment]
Fair enough. I'm not 100% sold on some of those arguments, but I appreciate that Standard is an opinionated, all-or-nothing style guide, so let's follow it. 👍 

This looks great, thanks for the effort. I'm keen on using the same exact approach on the [API repo](https://github.com/speedtracker/speedtracker-api).

File: package.json [hunk 1]
```diff
@@ -6,15 +6,19 @@
...
   },
+  "repository": "speedtracker/speedtracker",
   "author": "Eduardo Bouças <mail@eduardoboucas.com> (https://eduardoboucas.com)",
   "license": "MIT",
   "devDependencies": {
     "babel-loader": "^6.2.5",
     "babel-preset-es2015": "^6.13.2",
     "babel-preset-es2016": "^6.11.3",
     "babel-preset-react": "^6.11.1",
+    "eslint-plugin-standard": "^3.0.1",
```
Comment (feross) [inline package.json]: `eslint-plugin-standard` is not necessary if you're already installing `standard`. You can remove this!

Comment (ghost) [reply package.json]: Thanks. Thought I'd removed that after a copypasta job testing.

**Comment by ghost** [comment]
@eduardoboucas Remaining tasks completed, and feedback from @feross incorporated. I didn't see a `.travis.yml` so I didn't set that up. Please let me know if anything looks out of place. Thanks!

**Notes:**

- Commits have been squashed into a single feature
- Tested interface (default profile only) using data I received from `speedtracker-bot` [5 hours ago](https://github.com/jhabdas/speedtracker/commit/ed2b30fa8421106174168cffdd10dd6ab2f0648b)

**Comment by eduardoboucas** [comment]
Thanks @jhabdas! Looks great to me. Are you happy for me to merge it?

**Comment by ghost** [comment]
Bombs away!

**Comment by eduardoboucas** [comment]
🍾

**Pull request merged**